README.md

```diff
@@ -10,6 +10,7 @@ The snapshots are available in 2 formats: as MHTML files and as [WebTraversalLib
 The MHTML format is less lossy, a browser can render these pages though any Javascript on the page is lost.
 The WTL snapshots are produced by loading the MHTML pages into a chromium-based browser. To keep the WTL dataset more compact, they do not contain the screenshot of the rendered MTHML, but only its HTML, and page and element metadata with additional rendering information (bounding boxes of elements, font sizes etc.).
 For convenience, the datasets are provided with a train/test split in which no merchants in the test set are present in the training set.
+The `screenshot` dataset contains screeshots from a subset of snapshots (about 80% of the whole dataset) that are known to render correctly and are amenable to computer vision applications.
```

Comment:
You can mention the folder structure is identical as the WTL dataset and decompressing the screenshots in the same location puts the image in the WTL snapshot folder (in a sense, completing it).